build-tools/packages/build-cli/test/commands/generate/buildVersion.test.ts

```diff
@@ -160,7 +160,7 @@ describe("generate:buildVersion", () => {
             "--build",
             "12345",
             "--fileVersion",
-            "1.2.4",
+            "1.2.5",
```

Comment:
Just curious, why was this change necessary?

Reply:
It technically wasn't. I was trying to resolve test failures but this wasn't the root cause. I'm just leaving it since the test is still valid.

Reply:
Actually reverted this after all.